packages/xo-server-test/src/backupNg/backupNg.spec.js

```diff
@@ -66,6 +66,8 @@ const validateOperationTask = (task, props) => {
   })
 }
 
+// Note: on testing multiple backup Execution on the same VM, you need to bypass the check
+// of the VDI chains by setting 'bypassVdiChainsCheck' to true. See https://xen-orchestra.com/docs/backup_troubleshooting.html#vdi-chain-protection
```

Comment:
I think this clearer, do you agree?

```js

// Note: `bypassVdiChainsCheck` must be enabled because the XAPI might be able
// to coalesce VDIs as fast as the tests run.
//
// See https://xen-orchestra.com/docs/backup_troubleshooting.html#vdi-chain-protection
```

Reply:
Yes :+1: